### PR TITLE
WIP: discussion of changes required for build from source DO NOT MERGE

### PR DESCRIPTION
**Pull request by mgianatagh**
WIP: discussion of changes required for build from source DO NOT MERGE

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# .dockerignore
```
Comment (markxnelson) [inline .dockerignore:1]: need copyright and license

File: tests/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2020, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+FROM python:3
```
Comment (markxnelson) [inline tests/Dockerfile:4]: oracle linux

File: .dockerignore (hunk 1 above)
Comment (mgianatagh) [reply .dockerignore:1]: fixed

**Comment by mgianatagh** [comment]
Changes made for review feedback have been merged into oracle-build-from-source branch

**Pull request closed without merging**